src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,47 @@
+
+.container {
```

Comment:
The container class is not related to stars block and should be moved to a separate file or renamed using BEM naming conventions.